orchestrator-salt/src/main/resources/salt/pillar/top.sls

```diff
@@ -1,5 +1,11 @@
 base:
   '*':
-    - ambari.repo
     - consul.init
+
+  'roles:ambari_server':
```

Comment:
This is not correct. Ambari repo shall be '*' by default since every instance shall get the ambari repo info.